### PR TITLE
Fix redirection issues

### PR DESCRIPTION
**Pull request by medardm**

### CODE REVIEW

**Review by harveyjavier — APPROVED** [review]
Redirected!

File: src/App.tsx [hunk 1]
```diff
@@ -16,22 +16,22 @@ const App: React.FC = () => {
...
+            <Route path="/projects/:ProjectAddress" component={Project} />
 
-            {/* <Route path="/launch/staking" exact>
+            {/* <Route path="/staking" exact>
               <Staking />
             </Route> */}
             {/* Redirects */}
             <Route path="/" exact>
-                <Redirect to="/launch" />
+                <Redirect to="/" />
```
Comment (kojiadrianojr) [inline src/App.tsx:32]: Pwede na po ata to iclear kuys since no need naman.

**Review by swengr-janan — APPROVED** [review]
One direction!

**Review by kojiadrianojr — APPROVED** [review]
GNYP

**Review by mativanarquero — APPROVED** [review]

**Pull request merged**